README.md

```diff
@@ -46,7 +51,62 @@ git clone https://github.com/IBM/monitor-ibm-cloud-pak-with-watson-openscale
 cd monitor-ibm-cloud-pak-with-watson-openscale
 ```
 
-## 2. Configure OpenScale in a Jupyter Notebook
+## 2. Create a new project and deployment space
+### Create a New project
+
+In Cloud Pak for Data, we use the concept of a project to collect / organize the resources used to achieve a particular goal (resources to build a solution to a problem). Your project resources can include data, collaborators, and analytic assets like notebooks and models, etc.
+
+* Go the (☰) menu and click *Projects*
+
+![(☰) Menu -> Projects](doc/source/images/cpd-projects-menu.png)
+
+* Click on *New project*
+
+![Start a new project](doc/source/images/cpd-new-project.png)
+
+* We are going to create an empty project, as opposed to creating one from an existing file. Select the _*Create an empty project*_ option:
+
+![Create project from file](doc/source/images/cpd-create-empty-project.png)
+
+* Give the project a name and click `Create`:
+
+![Browse for project files](doc/source/images/cpd-new-project-create.png)
+
+### Create a Deployment Space
+
+Cloud Pak for Data uses the concept of `Deployment Spaces` to configure and manage the deployment of a set of related deployable assets. These assets can be data files, machine learning models, etc.
+
+* Go the (☰) menu and click `Analyze` -> `Analytics deployments`:
+
+![(☰) Menu -> Analytics deployments](doc/source/images/ChooseAnalyticsDeployments.png)
+
+* Click on `+ New deployment space`:
+
+![Add New deployment space](doc/source/images/addNewDeploymentSpace.png)
+
+* Give your deployment space a unique name, optional description, then click `Create`. You will use this space later when you deploy a machine learning model.
+
+#### Add collaborator
+Next, we will add a collaborator to the new deployment space, so that assets we deploy can be monitored in step 4.
+
+* Click on your new deployment space.
+
+![Select deployment space](doc/source/images/selectNewDeploymentSpace.png)
+
+* Click on the `Access control` tab and then click on `Add collaborators` on the right.
+
+![Deployment space access control](doc/source/images/deploymentSpaceAccessControl.png)
+
+* Enter "admin" as a Collaborator and select the user from the drop down list. Then click on the `Add to list` button.
+
+> **NOTE:** We are adding the user that configured the machine learning instance for OpenScale monitoring. In this case, the user is the admin user.
```

Comment:
This language is for the workshops, where we are fairly certain that the user who configurred ML for OpenScale is named `Admin`. For the notebook pattern, we should probably phrase it:
"If the user is someone other than `Admin`, then that user should be added as a collaborator here."

Reply:
Do you mean, instead of the **NOTE**?

Reply:
I'll just add a separate patch.